app/sign.js

```diff
@@ -421,20 +421,18 @@ const parseKey = co(function *(rawkey, coin, path) {
       throw new Error('Invalid mnemonic');
     }
     const seed = yield bip39.mnemonicToSeed(mnemonic);
-    let node = utxoLib.HDNode.fromSeedBuffer(seed);
+    let node = bip32.fromSeed(seed);
     if (path) {
       node = node.derivePath(path);
     }
     const xprv = node.toBase58();
     return xprv;
 
   }
-  // if it doesn't have commas, we expect it is an xprv or xlmsecret properly formatted
   if (path) {
-    let node = utxoLib.HDNode.fromPrivateKeyBuffer(Buffer.from(rawkey, 'hex'));
-    node = node.derivePath(path);
-    return node.toBase58();
+    throw new Error(`cannot derive key with path`)
```

Comment:
The related commit has a very nice comment: "There is no plausible use case where we want to instantiate a bip32 node from just the private key buffer (without chaincode)." I think something to that that effect (or even just the same exact wording) in a comment here would be helpful so we know why we can't derive with a path here.